src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2972,6 +2972,10 @@ def create_function(cmd, resource_group_name, name, storage_account, plan=None,
         snake_case_prop = _convert_camel_to_snake_case(prop)
         setattr(site_config, snake_case_prop, value)
 
+    # temporary workaround for dotnet-isolated linux consumption apps
+    if is_linux and consumption_plan_location is not None and runtime == 'dotnet-isolated':
```

Comment:
lower() before comparasion?

Reply:
Yeah that's a good catch but `runtime` is actually used a lot for comparisons in this method so I just did a blanket conversion here: https://github.com/Azure/azure-cli/blob/a726dc80e02bc1ce2937f0657519ed546a11343d/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L2853-L2854